src/Microsoft.AspNet.Mvc.Core/Controller.cs

```diff
@@ -1198,7 +1198,7 @@ public virtual Task<bool> TryUpdateModelAsync<TModel>([NotNull] TModel model)
         [NonAction]
         public virtual bool TryValidateModel([NotNull] object model)
         {
-            return TryValidateModel(model, prefix: null);
+            return TryValidateModel(model, prefix: string.Empty);
```

Comment:
@harshgMSFT we treat `null` and empty `string`s identically because it only serves to confuse if they have different meanings.  if there is code exposed to users where `null` and empty have different meanings, that code likely needs to be fixed or (in a few cases) _extremely_ well-documented.